### ISSUE TITLE
Hour Intervals

### ISSUE THREAD

**Issue by ghadeerrahhal** [twinssbc/Ionic-Calendar#65]
Is it possible to have the hours display in 15 or 30 minute intervals in either the day or week view?

**Comment by twinssbc** [comment]
@ghadeerrahhal Yes, check the **step** option in README.

**Comment by ghadeerrahhal** [comment]
I tried step but with no luck, I still see the times in 1 hour intervals, attached is a jsfiddle. Am I not using it properly?

https://jsfiddle.net/ghadeer/o7svcswt/2/

**Comment by twinssbc** [comment]
@ghadeerrahhal Actually in your jsfiddle link, the event is displayed correctly. If the event starts at 10:00, ends at 10:14, it will only occupy the quarter of the cell. Could you double check?

**Comment by ghadeerrahhal** [comment]
Yes it's displayed perfectly, sorry for not being clearer. Is it possible to have the times display 

7:00AM
7:15AM
7:30AM
7:45AM
8:00AM

etc?

**Comment by twinssbc** [comment]
@ghadeerrahhal you mean change the display in the first column? 
That's a bit difficult.
In week.html, below html code controls how to display the time.
You can change replace "::dt.date| date:  formatWeekViewDayHeader" with a method, then you can return any text you want.

```
<tr>
      <th class="calendar-hour-column"></th>
     <th class="weekview-header text-center" ng-repeat="dt in view.dates">{{::dt.date| date:  formatWeekViewDayHeader}}
     </th>
</tr>
```

**Comment by Aswin7955** [comment]
@twinssbc  I am trying to change the hour duration in day and weekly view. I have tried step. but not working.
Here is the code i have tried.

<calendar style="padding-top:2px;" ng-model="calendar.currentDate" calendar-mode="calendar.mode" event-source="calendar.eventSource"
                        range-changed="reloadSource(startTime, endTime)", 
                        event-selected="onEventSelected(event)" title-changed="onViewTitleChanged(title)"
                        time-selected="onTimeSelected(selectedTime, events, disabled)" step="15" format-hour-column="HH:mm">

Please help me out with this. Thanks in advance

**Comment by Krutagna** [comment]
I am having the same trouble. can someone provide the solution?

**Comment by twinssbc** [comment]
@Krutagna Could you provide your code?
